packages/bento-design-system/src/index.ts

```diff
@@ -119,3 +119,6 @@ export { useComponentsShowcase } from "./useComponentsShowcase";
 export type { ClassValue } from "clsx";
 
 export { Time } from "@internationalized/date";
+
+export { SSRProvider } from "@react-aria/ssr";
+export type { SSRProviderProps } from "@react-aria/ssr";
```

Comment:
This is the same approach as react-spectrum. I don't think we need necessarily need this export, but in case someone composes their provider differently, I think it makes sense to expose it.